### PR TITLE
bibtex: implements bibtex template

### PR DESCRIPTION
**Pull request by ioannistsanaktsidis**
Signed-off-by: Ioannis Tsanaktsidis <ioannis.tsanaktsidis@cern.ch>

### CODE REVIEW

**Comment by jalavik** [comment]
@ioannistsanaktsidis Thanks. The `bibtex` jinja filter is already in our codebase right?

File: inspire/base/templates/format/record/Inspire_BIBTEX.tpl [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{#
+# This file is part of Inspire.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
```
Comment (jalavik) [inline inspire/base/templates/format/record/Inspire_BIBTEX.tpl]: Can you change this to `INSPIRE` everywhere in this file?

**Comment by ioannistsanaktsidis** [comment]
@jalavik  bibtex filter is already in  our codebase. It is in /formatter_jinja_filters/record.py .

**Comment by ioannistsanaktsidis** [comment]
Just pushed the corrections!

**Comment by jalavik** [comment]
Thanks. Merged as eb66c84

**Pull request closed without merging**